package.json

```diff
@@ -12,13 +12,14 @@
   "private": true,
   "dependencies": {
     "@bufbuild/connect-web": "^0.6.0",
```

Comment:
Is looks like this is now called `@bufbuild/connect-es`.
https://github.com/bufbuild/connect-es/releases/tag/v0.8.0

Reply:
Please review the release notes because there are a few breaking changes documented there.

Reply:
@beeme1mr `https://www.npmjs.com/package/@bufbuild/connect-es` doesn't exist. I think the resources you are linking are just about protoc generation. As the link you supplied says:

>  In the future, @bufbuild/connect-web **will only export the Connect and gRPC-web transports.**

That's what we're concern about here - the exported transports.